code/game/objects/items/devices/transfer_valve.dm

```diff
@@ -76,21 +76,21 @@
 
 /obj/item/device/transfer_valve/Topic(href, href_list)
 	..()
-	if ( usr.stat || usr.restrained() )
+	if(!usr.canUseTopic())

```

Comment:
do you need to set be_close to true or is it on by default I don't recall I am a brainlet

Reply:
HELLO

Reply:
SORRY

Reply:
`
/mob/living/carbon/human/canUseTopic(atom/movable/M, be_close = 0)
	if(incapacitated() || lying )
		return
	if(!Adjacent(M) && (M.loc != src))
		if((be_close == 0) && (dna.check_mutation(TK)))
			if(tkMaxRangeCheck(src, M))
				return 1
		return
	return 1
`